Webex Connect Flows/How to manually upgrade from v3.2 to v3.3 workflows.MD

```diff
@@ -0,0 +1,21 @@
+
```

Comment:
3.2 folder also needs to be deleted, isn't it?

Reply:
I have deleted 3.2 completely what's the problem you see with this file Shruthi?

Reply:
I just commented here as i could not get a reference to a 3.2 file. Ignore the location of the comment. In the files changed section, I don't see the removal of 3.2 folder, hence asked

Reply:
It's deleted that's why showing so many file changes :)